universal-login-react/test/ui/UDashboard.tsx

```diff
@@ -0,0 +1,39 @@
+import {expect} from 'chai';
+import {mount} from 'enzyme';
+import React from 'react';
+import {getWallets, createMockProvider} from 'ethereum-waffle';
+import {setupSdk, createWallet} from '@universal-login/sdk/testutils';
+import {DeployedWallet} from '@universal-login/sdk';
+import {UDashboard} from '../../src/ui/UFlow/UDashboard';
+import {waitExpect} from '@universal-login/commons';
+import {Wallet, utils} from 'ethers';
+
+describe('INT: UDashboard', () => {
+  let wallet: Wallet;
+  let deployedWallet: DeployedWallet;
+  let contractAddress: string;
+  let privateKey: string;
+  const ensName = `jarek.mylogin.eth`;
+  const initialAmount = `1.99`;
+
+  beforeEach(async () => {
+    ([wallet] = await getWallets(createMockProvider()));
+    const {sdk} = await setupSdk(wallet, '33113');
+    sdk.priceObserver.getCurrentPrices = () => {
+      return Promise.resolve({ETH: {USD: 1, DAI: 1, ETH: 1}});
```

Comment:
let's make it a bit more intuitive, e.g.:
`return Promise.resolve({ETH: {USD: 100, DAI: 99, ETH: 1}});`